gst/nnstreamer/tensor_filter/tensor_filter.c

```diff
@@ -989,8 +989,11 @@ _gst_tensor_filter_transform_update_outbuf (GstBaseTransform * trans,
   /* If output combination is defined, append input tensors first */
   if (priv->combi.out_combi_i_defined) {
     for (list = priv->combi.out_combi_i; list != NULL; list = list->next) {
+      GstTensorInfo combi_info;
       i = GPOINTER_TO_UINT (list->data);
 
+      gst_tensor_info_init (&combi_info);
```

Comment:
gst_tensor_info_init (&combi_info);
_info = gst_tensors_info_get_nth_info (&priv->in_config.info, i);

2nd:
gst_tensor_meta_info_convert (&in_trans_data->meta[i], &combi_info);
_info = &combi_info;

3rd
unnecessary

final appending
gst_tensor_buffer_append_memory (outbuf, mem, _info);
gst_tensor_info_free (&combi_info);

Reply:
This method makes the code more intuitive to understand. Thanks.